.devcontainer/devcontainer.json

```diff
@@ -11,6 +11,7 @@
     "PATH": "/workspace/bin:${containerEnv:PATH}"
   },
   "postCreateCommand": "bundle install && npm install",
+  "postStartCommand": "bin/setup && bin/dev",
```

Comment:
I don't think we should couple the devcontainer lifecycle directly with the app process management (i.e. `bin/dev`).  

We could probably throw `bin/setup` at the end of the `postCreateCommand` to make things a bit easier on initial setup, but I think users should run `bin/dev` manually.